packages/design-system/src/components/Promo/style.scss

```diff
@@ -0,0 +1,156 @@
+@import '../../scss/variables/colors';
+
+$base-class: 'promo';
+
+.#{$base-class} {
+  position: relative;
+  padding: 20px;
+  overflow: hidden;
+  word-break: break-word;
+  width: 100%;
+  box-sizing: border-box;
+  background-color: #f3f7f9;
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  min-width: 180px;
+
+  &--light {
+    border: solid 1px #dde2e6;
+    background-color: #ffffff;
+  }
+
+  &--small {
+    max-width: 400px;
+    flex-wrap: wrap;
+
+    .#{$base-class}__content {
+      margin-bottom: 12px;
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+      flex-direction: column;
+
+      &-img {
+        width: 48px;
+        height:48px;
+        margin-bottom: 12px;
+      }
+      &-wrapper {
+        display: flex;
+        flex-direction: column;
+      }
+      &-header {
+        font-weight: 600;
+      }
+      &-footer {
+        display: flex;
+
+        &-button {
+          margin-right: 24px;
+        }
+      }
+    }
+  }
+
+  &--medium {
+    max-width: 800px;
+    flex-wrap: wrap;
+
+    .#{$base-class}__content {
+      margin-bottom: 12px;
+      display: flex;
+      justify-content: flex-start;
+      flex-direction: row;
+
+      &-img {
+        width: 48px;
+        height:48px;
+        margin-right: 16px;
+      }
+      &-wrapper {
+        display: flex;
+        flex-direction: column;
+        max-width: 520px;
+      }
+      &-header {
+        font-weight: 600;
+      }
+      &-footer {
+        margin: 0 56px 0 56px;
+        display: flex;
+        flex-shrink: 0;
+        
+        &-button {
+          margin-right: 24px;
+        }
+      }
+    }
+  }
+
+  &--large {
+    .#{$base-class}__content {
+      margin-bottom: 12px;
+      display: flex;
+      justify-content: flex-start;
+      flex-direction: row;
+
+      &-img {
+        width: 48px;
+        height:48px;
+        margin-right: 19px;
```

Comment:
Was it 19px on the design?

Reply:
yes, but changed to 20px - needs design review